libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/LoginActivity.kt

```diff
@@ -339,6 +339,11 @@ open class LoginActivity : FragmentActivity() {
     override fun onResume() {
         super.onResume()
         wasBackgrounded = false
+
+        // If debug LoginOptions were changed reload the webview.
+        if (SalesforceSDKManager.getInstance().isDebugBuild && webView.url != viewModel.loginUrl.value) {
+            viewModel.reloadWebView()
+        }
```

Comment:
If anyone has a better was to implement this I am all for it.  Unfortunately, calling reload from `LoginOptionsActivity` does not update the webview (because it is not currently rendered, I assume).

Reply:
Yes.  If WebServer Flow is used getting a new authorization url cause the url to change because code verifier will be new.  If UserAgent flow is used we set the URL to about_blank before getting a "new" authorization url (which will be identical to the old one if nothing else changes) to force the reload.  

It is just annoying I have to call it here also.

Reply:
But if web server flow is not being used then loginUrl won't change.
On iOS, I restart authentication when coming out of showing the login options: https://github.com/forcedotcom/SalesforceMobileSDK-iOS/pull/3950/files#diff-1334e2942ea2928b6db7bca76fdebd960b717bc0463c404244b998a86ee08121R494